### ISSUE TITLE
Wrong config filename

### ISSUE THREAD

**Issue by freehck** [vinser/flibgolite#13]
```
/flibgolite # ./flibgolite -config
Default config file "./config/flibgolite.yml" was created for customization
/flibgolite # ls ./config/flibgolite.yml
ls: ./config/flibgolite.yml: No such file or directory
/flibgolite # ls config/
config.yml
```

**Comment by freehck** [comment]
tested on tag v1.0.0

**Comment by vinser** [comment]
It's wrong message after config file creation for customization purposes

**Comment by vinser** [comment]
fixed in v1.0.1
